bin/release.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+import {fileURLToPath} from 'url'
+import {dirname, join} from 'pathe'
+import {createRequire} from 'module'
+import {readFileSync} from 'fs'
+
+const require = createRequire(import.meta.url)
+const {Octokit} = require('@octokit/rest')
+const execa = require('execa')
+
+const octokit = new Octokit({
+    auth: process.env.GITHUB_TOKEN,
+});
+const goos = ['linux', 'windows', 'darwin']
+const goarch =  ['386', 'amd64']
```

Comment:
Nitpick: Since they are the collection of supported OSs and Architectures, what about naming these `supportedOSs` and `supportedArchitectures`?